contracts/HubPool.sol

```diff
@@ -40,6 +40,7 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
         uint64 unclaimedPoolRebalanceLeafCount;
         bytes32 poolRebalanceRoot;
         bytes32 destinationDistributionRoot;
+        bytes32 slowRelayDataRoot;
```

Comment:
you'll need to add this to `initiateRelayerRefund`. I can do that in a separate PR though, If you'd like as this will break a bunch of existing tests.

Reply:
I dont like this variable name. can it be `SlowRelayFulfilmentRoot`?

Reply:
Oh, I thought I added it everywhere, will do another pass to better understand where I missed.

On the naming, sure!

Reply:
Done on both.